app/views/account/admin/issues/_issue.html.slim

```diff
@@ -4,7 +4,7 @@ tr
   td= issue.user.full_name
   td
     - unless issue.pending?
-      = show_issue_status(issue)
+      = get_stylized_status_style(issue.status, issue.status_name)
```

Comment:
something I dont like in this method :) but can't tell what exactly. for now lets consider it's as OK